etna/models/moving_average.py

```diff
@@ -8,6 +8,11 @@ class MovingAverageModel(SeasonalMovingAverageModel):
         y_{t} = \\frac{\\sum_{i=1}^{n} y_{t-i} }{n},
 
     where :math:`n` is window size.
+
+    Notes
+    -----
+    This model supports in-sample and out-of-sample prediction decomposition.
```

Comment:
> This model supports in-sample and out-of-sample prediction decomposition.

I'm not sure that it is an implementation detail and should be in Notes.

> Prediction components are corresponding target lags with weights of 1/window

May it could be better to write :math:`1 / window`